docs/troubleshooting_pi-server_integration.rst

```diff
@@ -0,0 +1,180 @@
+.. Images
+.. |img_001| image:: images/tshooting_pi_001.jpg
+.. |img_002| image:: images/tshooting_pi_002.jpg
+.. |img_003| image:: images/tshooting_pi_003.jpg
+.. |img_004| image:: images/tshooting_pi_004.jpg
+.. |img_005| image:: images/tshooting_pi_005.jpg
+.. |img_006| image:: images/tshooting_pi_006.jpg
+.. |img_007| image:: images/tshooting_pi_007.jpg
+.. |img_008| image:: images/tshooting_pi_008.jpg
+.. |img_009| image:: images/tshooting_pi_009.jpg
+.. |img_010| image:: images/tshooting_pi_010.jpg
+
+Troubleshooting the PI-Server integration
+=========================================
+
+This section describes how to trouble shoot issues with the PI-Server integration
+using Fledge version >= 1.9.x and PI Web API 2019 SP1 1.13.0.6518
+
+- Log files
+- How to check the PI Web API is installed and running
+- Commands to check the PI Web API
+- Error messages and the related causes
+- Possible solutions to common situations
+
+Log files
+---------
+
+Fledge logs into the system syslog, mainly warnings and errors and in some circumstances rows of severity information.
+The name of the north instance should be used to extract just the logs about the PI-Server integration, as in this example:
+
+screenshot from the Fledge GUI
+
+|img_003|
+
+.. code-block:: console
+
+    $ sudo cat /var/log/syslog | grep North_Readings_to_PI
+
+sample message:
+
+    user.info, 6,1,Mar 15 08:29:57,localhost,Fledge, North_Readings_to_PI[15506]: INFO: SendingProcess is starting
+
+another sample message:
+
+    North_Readings_to_PI[20884]: WARNING: Error in retrieving the PIWebAPI version, The PI Web API server is not reachable, verify the network reachability
+
+How to check the PI Web API is installed and running
+----------------------------------------------------
+
+Open the URL *https://piserver_1/piwebapi* in the browser, substituting *piserver_1* with the name/address of your PI Server, to
+verify the reachability and proper installation of PI Web API.
+If PI Web API is configured for *Basic* authentication a prompt asking user name/password like the following one will appear:
+
+|img_002|
+
+**NOTE:**
+
+- *The same user name/password configured in Fledge in the north instance should be used.*
```

Comment:
Enter the user name and password which you set in your Fledge configuration.

Reply:
done